src/ocnicepost.fd/outputvars.F90

```diff
@@ -0,0 +1,146 @@
+    ii = ii + 1; outvars(ii)%input_var_name  = 'SW'        ; outvars(ii)%var_remapmethod  = 'conserve'
+    ii = ii + 1; outvars(ii)%input_var_name  = 'LW'        ; outvars(ii)%var_remapmethod  = 'conserve'
+    ii = ii + 1; outvars(ii)%input_var_name  = 'evap'      ; outvars(ii)%var_remapmethod  = 'conserve'
+    ii = ii + 1; outvars(ii)%input_var_name  = 'lprec'     ; outvars(ii)%var_remapmethod  = 'conserve'
+    ii = ii + 1; outvars(ii)%input_var_name  = 'fprec'     ; outvars(ii)%var_remapmethod  = 'conserve'
+    ii = ii + 1; outvars(ii)%input_var_name  = 'LwLatSens' ; outvars(ii)%var_remapmethod  = 'conserve'
+    ii = ii + 1; outvars(ii)%input_var_name  = 'Heat_PmE'  ; outvars(ii)%var_remapmethod  = 'conserve'
+
+    ! 2D vector states on stagger locations; remapped bilinearly
+    ii = ii + 1; outvars(ii)%input_var_name  = 'SSU'
+                 outvars(ii)%var_grid = 'Cu'
+                 outvars(ii)%var_pair = 'SSV'
+                 outvars(ii)%var_pair_grid = 'Cv'
+
+    ii = ii + 1; outvars(ii)%input_var_name  = 'SSV'
+                 outvars(ii)%var_grid = 'Cv'
+                 outvars(ii)%var_pair = 'SSU'
+                 outvars(ii)%var_pair_grid = 'Cu'
+
+    ! 2D vector fluxes on stagger locations; remapped conservatively
+    ii = ii + 1; outvars(ii)%input_var_name = 'taux'
+                 outvars(ii)%var_grid = 'Cu'
+                 outvars(ii)%var_pair = 'tauy'
+                 outvars(ii)%var_pair_grid = 'Cv'
+                 outvars(ii)%var_remapmethod = 'conserve'
+
+    ii = ii + 1; outvars(ii)%input_var_name = 'tauy'
+                 outvars(ii)%var_grid = 'Cv'
+                 outvars(ii)%var_pair = 'taux'
+                 outvars(ii)%var_pair_grid = 'Cu'
+                 outvars(ii)%var_remapmethod = 'conserve'
+
+    ! 3D scalars with native grid location on cell centers; remapped bilinearly
+    ii = ii + 1; outvars(ii)%input_var_name  = 'temp' ; outvars(ii)%var_dimen = 3
+    ii = ii + 1; outvars(ii)%input_var_name  = 'so'   ; outvars(ii)%var_dimen = 3
+
+    ! 3D vectors on stagger locations; remapped bilinearly
+    ii = ii + 1; outvars(ii)%input_var_name  = 'uo'
+                 outvars(ii)%var_grid = 'Cu'
+                 outvars(ii)%var_pair = 'vo'
+                 outvars(ii)%var_pair_grid = 'Cv'
+                 outvars(ii)%var_dimen = 3
+
+    ii = ii + 1; outvars(ii)%input_var_name  = 'vo'
+                 outvars(ii)%var_grid = 'Cv'
+                 outvars(ii)%var_pair = 'uo'
+                 outvars(ii)%var_pair_grid = 'Cu'
+                 outvars(ii)%var_dimen = 3
+
+    ! set default output name
+    outvars(:)%output_var_name = outvars(:)%input_var_name
+
+  end subroutine ocnvars_typedefine
+
+  subroutine icevars_typedefine
+
+    ! local variables
+    integer :: ii = 0
+
+    !set defaults
+    outvars(:)%input_var_name=''
+    outvars(:)%var_grid  = 'Ct'
+    outvars(:)%var_remapmethod  = 'bilinear'
```

Comment:
What happens if a variable requires a different type of interpolation type (e.g., nearest-neighbor)? This should not be hard coded.

Reply:
State variables should be remapped with linear interpolation. Fluxes could be remapped with nearest neighbor. Is this something desired for MOM6 or CICE6 remapped fields?

Reply:
@DeniseWorthen What happens if an additional variable is needed in the future and requires nearest-neighbor interpolation? How do you do this without needing to rebuild the entire application?

Reply:
@HenryWinterbottom-NOAA We can address the customization as an improvement activity.
For now, let us focus on this improved functionality that can be leveraged